modules/compliance-scheduling-pods-with-resource-requests.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+:_content-type: CONCEPT
+[id="compliance-scheduling-pods-with-resource-requests"]
+= Scheduling Pods with resource requests
+
+When a Pod is created, the scheduler selects a Node for the Pod to run on. Each node has a maximum capacity for each resource type in the amount of CPU and memory it can provide for the Pods. The scheduler ensures that the sum of the resource requests of the scheduled containers is less than the capacity nodes for each resource type.
+
+Although memory or CPU resource usage on nodes is very low, the scheduler might still refuse to place a Pod on a node if the capacity check fails to protect against a resource shortage on a node.
+
+For each container, you can specify resource limits and requests, including the following:
+
+[source,terminal]
+----
+spec.containers[].resources.limits.cpu
+spec.containers[].resources.limits.memory
+spec.containers[].resources.limits.hugepages-<size>
+spec.containers[].resources.requests.cpu
+spec.containers[].resources.requests.memory
+spec.containers[].resources.requests.hugepages-<size>
+----
+
+[NOTE]
+====
+Although you can only specify requests and limits for individual containers, it is also useful to consider the overall resource requests and limits for a Pod. For a particular resource, a Pod resource request or limit is the sum of the resource requests or limits of that type for each container in the Pod.
+====
+
+.Example Pod resource requests and limits
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: frontend
+spec:
+  containers:
+  - name: app
+    image: images.my-company.example/app:v4
+    resources:
+      requests: <1>
+        memory: "64Mi"
+        cpu: "250m"
+      limits: <2>
+        memory: "128Mi"
+        cpu: "500m"
+  - name: log-aggregator
+    image: images.my-company.example/log-aggregator:v6
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
+----
+<1> The Pod is requesting 64Mi of memory and 250 millicores.
```

Comment:
Please be consistent with abbreviating and not abbreviating units in the same sentence. Also, you need a noun after "250 millicores"